UNCODE/UNCODE/Common/Views/Cells/MetabolicTypeCell/MetabolicTypeTableViewCell.swift

```diff
@@ -25,12 +28,22 @@ class MetabolicTypeTableViewCell: UITableViewCell {
     override func layoutSubviews() {
         super.layoutSubviews()
         contentView.frame = contentView.frame.inset(by: UIEdgeInsets(top: 0, left: 20, bottom: 0, right: 20))
-        contentView.layer.cornerRadius = 10
-        contentView.layer.masksToBounds = true
+        contentView.layer
```

Comment:
this can go into `awakeFromNib`, ideally create a `commonInit` method and call it from `awakeFromNib`